### DISCUSSION TITLE
spaceship update_availability method raise Not Found error

### DISCUSSION THREAD

**Discussion by schunlee** [fastlane/fastlane#17699] [Q&A]
[https://github.com/fastlane/fastlane/issues/12283](https://github.com/fastlane/fastlane/issues/12283)
`Spaceship::Tunes.login($FASTLANE_USER, $FASTLANE_PASSWORD)`
`        app = Spaceship::Application.find(ENV['APP_IDENTIFIER'])`
`        availability = app.availability`
`        availability.cleared_for_preorder = pre_order_flag`
`        if pre_order_flag.to_s == "false"`
`            availability.app_available_date = nil`
`        else`
`            availability.app_available_date = pre_order_date`
`        end`
`        puts app.update_availability!(availability)`
these days, I find update_availability! not works again, raise **Not Found** Error. Please someone can help me.

**Comment by rogerluan** [comment]
Hey @schunlee 👋 

Please fill out a new issue, filling all the info asked: https://github.com/fastlane/fastlane/issues/new/choose

**Reply by schunlee** [reply]
Hi, @rogerluan I have created one, here [https://github.com/fastlane/fastlane/issues/17763](https://github.com/fastlane/fastlane/issues/17763)
but I still don't know how to update pre_order status via spaceship, please help me, thanks very much.
